cruise/CruiseInformation.csv

```diff
@@ -311,5 +311,6 @@ EK20230131,Elakha,2023-01-31T16:29:00,2023-01-31T22:10:00,Endurance CSPP deploym
 AR71-04,Neil Armstrong,2023-02-21T15:08:00,2023-03-01T13:26:00,AST3 test mooring deployments at Pioneer MAB
 SKQ202304S,Sikuliaq,2023-03-07T20:10:00,2023-03-21T21:00:00,Endurance 18 moorings profilers and gliders
 TS20230310,Tenacious II,2023-03-10T10:00:00,2023-03-10T23:00:00,ASMAT1 test glider deployments
+OD20230409,Oscar Dyson,2023-04-09T00:00:00,2023-04-16T23:59:00,Papa glider gp_363 recovery
```

Comment:
good

Reply:
We should be using the cruise numbers that the ship operator uses (which per Allen Smith is in a DY23## format).  Because we have to use a different code (OD instead of DY), then using the format that we use for ships that don’t have cruise numbers makes sense.